src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -766,10 +776,10 @@ def __convert_feature_dict_to_keyvalue_list(features_dict, enabled_for_keyword):
                                 break
 
                             filter_param = val.get("parameters", {})
-                            new_val = {'name': val["name"]}
+                            new_val = {FeatureFlagConstants.FILTER_NAME: val["name"]}
```

Comment:
RHS can also use constants for `name` and `parameters`

Reply:
The idea was to differentiate between the key names in the feature dictionary from the file and the actual property names in the feature flag value. But since they are the same, it might actually be confusing to explicitly specify one over the other. This has been updated, thanks!